test/e2e/zeroXtrading.ts

```diff
@@ -0,0 +1,448 @@
+// // mainnet fork for 0x trading through 0xCallee
+// import {
+//   PermitCalleeInstance,
+//   Trade0xInstance,
+//   MockERC20Instance,
+//   OtokenInstance,
+//   WETH9Instance,
```

Comment:
should we remove this file for now and add it back later once it passes?

Reply:
remove this.